lms/djangoapps/edraak_forus/tests/tests.py

```diff
@@ -0,0 +1,388 @@
+# pylint: disable=missing-docstring
+import json
```

Comment:
Try to group the imports.

 - Django
 - Native
 - edX
 - Edraak

What do you think?